### ISSUE TITLE
Slider keyboard controls do not work when it is only focusable element when using NVDA

### ISSUE THREAD

**Issue by jonathanglister** [angular/material2#12232]
#### Bug, feature request, or proposal:
If a `mat-slider` is the only focusable element on a page, and the user is using NVDA, using the arrow keys to change the value of the slider does not do anything.

#### What is the expected behavior?
I should be able to focus the slider and change its value with the arrow keys.

#### What is the current behavior?
If I focus the slider, I can't change its value with the arrow keys.

#### What are the steps to reproduce?
This doesn't work in Stackblitz because of the other elements influencing the tab order. I used a component with boilerplate CSS and TS files, with this (mycomponent).component.html:

`<div class="basic-container">

  
  <mat-card>
    <mat-card-content>
      <mat-slider min="1" max="100" thumbLabel></mat-slider>
    </mat-card-content>
  </mat-card>
</div>`

Once you have the component running:

1) Start NVDA
2) Open Chrome and navigate to the component webpage.
3) Press Tab to focus the slider.
4) Try using the arrow keys to change the slider value.

#### What is the use-case or motivation for changing an existing behavior?
The keyboard controls should work regardless of how many elements are on the page - this is a bug.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Angular: 6.3.3
Chrome: Version 67.0.3396.99 (Official Build) (64-bit)
OS: Windows 10 Enterprise, Version 1709, OS build 16299.547
TypeScript: 2.7.2
NVDA: 2018.2

#### Is there anything else we should know?
This may be an NVDA issue.

**Comment by crisbeto** [comment]
I can definitely reproduce it, but I'm not sure whether we can do much about it. The slider receives focus as expected, but NVDA seems to block the keyboard events from reaching it.
